lua/git-worktree/git.lua

```diff
@@ -145,6 +153,11 @@ function M.create_worktree_job(path, branch, found_branch)
         table.insert(worktree_add_args, branch)
     end
 
+    if found_upstream then
```

Comment:
Actually, it only makes sense to use --track when the branch does not exist and -b is used in git-worktree add.